actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -1691,6 +1693,17 @@ private void createUser(Request actorMessage) {
     response.put(JsonKey.ACCESSTOKEN, accessToken);
     sender().tell(response, self());
 
+    /*
+     * Create User Entity in Registry
+     */
+    if(ProjectUtil.getConfigValue("sunbird_open_saber_bridge_enable").equalsIgnoreCase("true")){
```

Comment:
Can we change condition as "true".equalsIgnoreCase(...)?

Reply:
Review comment incorporated in code